dradis/dradis-ce
# Add mentions/notifications to notes

### Spec
Since we're adding comments to the view, should also add mentions.

### Proposed solution
Add mentions to the notes with the mentionable users.

### How to test
1. Create a note in a project.
2. With user_1,  comment on the note with a mention to user_2
3. Check that user_2 gets a notification for that comment
